src/current/v23.2/read-committed.md

```diff
@@ -0,0 +1,928 @@
+
+{{site.data.alerts.callout_info}}
+`READ COMMITTED` on CockroachDB provides stronger isolation than `READ COMMITTED` on PostgreSQL. On CockroachDB, `READ COMMITTED` prevents anomalies within single statements. For complete details on how `READ COMMITTED` is implemented on CockroachDB, see the [Read Committed RFC](https://github.com/cockroachdb/cockroach/blob/master/docs/RFCS/20230122_read_committed_isolation.md).
+{{site.data.alerts.end}}
+
+## Enable `READ COMMITTED` isolation
+
+To make `READ COMMITTED` isolation available to use on a cluster, enable the following [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) in the SQL shell:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING sql.txn.read_committed_isolation.enabled = 'true';
+~~~
+
+After you enable the cluster setting, you can configure `READ COMMITTED` isolation at the [session](#set-the-current-session-to-read-committed) or [transaction](#set-the-current-transaction-to-read-committed) level.
+
+{{site.data.alerts.callout_info}}
+If the cluster setting is not enabled, `READ COMMITTED` transactions will run as `SERIALIZABLE`.
+{{site.data.alerts.end}}
+
+### Set the current session to `READ COMMITTED`
+
+To set all future transactions in a session to run at `READ COMMITTED` isolation, use one of the following options:
+
+- The [`SET SESSION CHARACTERISTICS`]({% link {{ page.version.version }}/set-vars.md %}#special-syntax-cases) statement:
+
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	SET SESSION CHARACTERISTICS AS TRANSACTION ISOLATION LEVEL READ COMMITTED;
+	~~~
+
+- The [`default_transaction_isolation`]({% link {{ page.version.version }}/session-variables.md %}#default-transaction-isolation) session variable:
+
+	At the database level:
+
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	SET default_transaction_isolation = 'read committed';
+	~~~
+
+	At the [role level]({% link {{ page.version.version }}/alter-role.md %}#set-default-session-variable-values-for-a-role):
+
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	ALTER ROLE foo SET default_transaction_isolation = 'read committed';
+	~~~
+
+- The `default_transaction_isolation` session variable as a [connection parameter]({% link {{ page.version.version }}/connection-parameters.md %}#connect-using-a-url) with [`cockroach sql`]({% link {{ page.version.version }}/cockroach-sql.md %}):
+
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	cockroach sql -–url='postgresql://{username}@{host}:{port}/{database}?options=-c default_transaction_isolation=read%20committed'
```

Comment:
Doesn't the following also work:
```
cockroach sql -–url='postgresql://{username}@{host}:{port}/{database}?default_transaction_isolation=read%20committed'
```

Reply:
I can't immediately tell, but `options=-c` seems consistent with our docs, so I'll link directly to this section: https://www.cockroachlabs.com/docs/v23.2/connection-parameters#supported-options-parameters